packages/fluentui/e2e/tests/toolbarMenuOverflow-example.tsx

```diff
@@ -1,45 +1,59 @@
 import React from 'react';
 import _ from 'lodash';
-import { Toolbar, ToolbarItem } from '@fluentui/react';
+import { Toolbar, ToolbarItem, ToolbarMenu, Ref, Button } from '@fluentui/react';
 
 export const selectors = {
   toolbarItem: ToolbarItem.className,
   toolbar: Toolbar.className,
-  toolbarItemWrapper: ToolbarItem.slotClassNames.wrapper
+  toolbarItemWrapper: ToolbarItem.slotClassNames.wrapper,
+  menuTrigger: 'menu-trigger',
+  itemButtonId: 'item-button',
+  toolbarMenu: ToolbarMenu.className,
+  afterToolbarId: 'after'
 };
 
 export const itemsCount = 20;
+const buttonAfterToolbarRef = React.createRef<HTMLButtonElement>();
 
 const ToolbarExampleOverflow = () => {
   const icons = ['bold', 'italic', 'underline'];
 
   const itemData = _.times(itemsCount, i => ({
     key: `b${i}`,
+    id: `${selectors.itemButtonId}-${i}`,
     content: `${icons[i % icons.length]} #${i}`,
     icon: icons[i % icons.length],
     title: `${icons[i % icons.length]} #${i}`,
-    // first hald of items are unwrapped, rest are wrapped.
-    ...(i >= itemsCount / 2 && { menu: [] })
+    onClick: i + 1 == itemsCount ? () => buttonAfterToolbarRef.current.focus() : undefined,
+    // first hald of items are unwrapped, rest are wrapped, expect of the last item
```

Comment:
// first half 

That was my comment while working a previous bug, can you fix it? :D

Reply:
sure :) done